AnywhereExamplePanel/CSXS/manifest.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ExtensionManifest Version="4.0" ExtensionBundleId="com.example.AnywherePanel" ExtensionBundleVersion="1.0.0"
+ExtensionBundleName="AnywherePanel" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<ExtensionList>
+		<Extension Id="com.example.AnywherePanel" Version="1.0" />
```

Comment:
it is best practice for the ExtensionId and ExtensionBundleId to be different. So I suggest changing ExtensionId to be com.example.AnywherePanel.extension